src/components/containers/ConnectedBorrowForm/ConnectedBorrowForm.tsx

```diff
@@ -150,8 +150,8 @@ const ConnectedBorrowForm: React.FunctionComponent<ConnectedBorrowFormProps> = (
       />
       <SwapInfo
         balance={form.selectedFixedDebt}
+        currentPositionMarginRequirement={form.margin}
```

Comment:
prop positionMargin just below has same value... why not use that prop?

Reply:
actually that's not even used for the borrow product; will delete